pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -2,6 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+/// The valid dash tool labels stored in the [DashTool] enum
+Set<String> get validDashTools => DashTool.values.map((e) => e.label).toSet();
```

Comment:
Given that this is only used in one place, consider inlining it.

And given that it's for display purposes, consider adding `.toList()..sort()` to make it easier for readers to find what they're looking for in the list.